sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -27,12 +27,12 @@
 import AzureCore
 import Foundation
 
-// swiftlint:disable function_body_length
+// swiftlint:disable function_body_length type_body_length file_length cyclomatic_complexity identifier_name
```

Comment:
Probably not :p But I wanted to keep refactoring to satisfy lint for a separate PR.

Reply:
Maybe it would be best to just not add the disable. Even if we address it sometime down the road, if we disable it now, we'll probably never come back to it.

Reply:
Resolved.